tbille/metacheck
# Package using poetry

# Summary

Prepare the code to be packaged.

- I am using poetry to test it out (https://python-poetry.org/)
- Refactor the app to be under ` ./metadata`

# QA

- Install poetry (https://python-poetry.org/docs/)
- Install dependencies: `poetry install`
- Run the project on a small site: ` poetry run metacheck https://lukewh.com`
- Should work as it used to :tada:
